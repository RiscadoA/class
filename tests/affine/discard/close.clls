proc main() {
    cut {
        println("before affine"); affine x; println("affine");
        println("close"); close x
        |x: ~affine close|
        println("discard"); discard x
    }
};;