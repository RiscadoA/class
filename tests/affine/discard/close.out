before affine
affine
close
discard
